workflows/epigenetics/ChIPseq/ChIPseq.ga

```diff
@@ -0,0 +1,654 @@
+{
+    "a_galaxy_workflow": "true",
+    "annotation": "This workflow take as input a collection of paired fastq. Remove adapters with cutadapt, map pairs with bowtie2. Keep MAPQ30 and concordant pairs. MACS2 for paired bam.",
+    "creator": [
+        {
+            "class": "Person",
+            "identifier": "https://orcid.org/0000-0002-1964-4960",
+            "name": "Lucille Delisle"
+        }
+    ],
+    "format-version": "0.1",
+    "license": "AGPL-3.0-or-later",
```

Comment:
I think we typically go with `MIT` for most Galaxy materials, any reason you prefer AGPL ?

Reply:
Not really, I can switch